tests/helpers/recompute_zones_helpers.test.ts

```diff
@@ -0,0 +1,885 @@
+    conf2 = ["C3:E5", "F6:G7"];
+    result = new Map<number, number[]>([
+      [0, []],
+      [2, [2, 5]],
+      [5, [5, 7]],
+      [7, []],
+    ]);
+    expect(computeProfiles(conf1, [])).toEqual(result);
+    expect(computeProfiles(conf2, [])).toEqual(result);
+
+    conf1 = ["G6:H7", "C3:E5"];
+    conf2 = ["C3:E5", "G6:H7"];
+    result = new Map<number, number[]>([
+      [0, []],
+      [2, [2, 5]],
+      [5, []],
+      [6, [5, 7]],
+      [8, []],
+    ]);
+    expect(computeProfiles(conf1, [])).toEqual(result);
+    expect(computeProfiles(conf2, [])).toEqual(result);
+
+    // ///////////////////////////////////////
+    // translation de 2x2 sur row 6 avec C3:E5
+    // ///////////////////////////////////////
+
+    conf1 = ["A7:B8", "C3:E5"];
+    conf2 = ["C3:E5", "A7:B8"];
+    result = new Map<number, number[]>([
+      [0, [6, 8]],
+      [2, [2, 5]],
+      [5, []],
+    ]);
+    expect(computeProfiles(conf1, [])).toEqual(result);
+    expect(computeProfiles(conf2, [])).toEqual(result);
+
+    conf1 = ["B7:C8", "C3:E5"];
+    conf2 = ["C3:E5", "B7:C8"];
+    result = new Map<number, number[]>([
+      [0, []],
+      [1, [6, 8]],
+      [2, [2, 5, 6, 8]],
+      [3, [2, 5]],
+      [5, []],
+    ]);
+    expect(computeProfiles(conf1, [])).toEqual(result);
+    expect(computeProfiles(conf2, [])).toEqual(result);
+
+    conf1 = ["C7:D8", "C3:E5"];
+    conf2 = ["C3:E5", "C7:D8"];
+    result = new Map<number, number[]>([
+      [0, []],
+      [2, [2, 5, 6, 8]],
+      [4, [2, 5]],
+      [5, []],
+    ]);
+    expect(computeProfiles(conf1, [])).toEqual(result);
+    expect(computeProfiles(conf2, [])).toEqual(result);
+
+    conf1 = ["D7:E8", "C3:E5"];
+    conf2 = ["C3:E5", "D7:E8"];
+    result = new Map<number, number[]>([
+      [0, []],
+      [2, [2, 5]],
+      [3, [2, 5, 6, 8]],
+      [5, []],
+    ]);
+    expect(computeProfiles(conf1, [])).toEqual(result);
+    expect(computeProfiles(conf2, [])).toEqual(result);
+
+    conf1 = ["E7:F8", "C3:E5"];
+    conf2 = ["C3:E5", "E7:F8"];
+    result = new Map<number, number[]>([
+      [0, []],
+      [2, [2, 5]],
+      [4, [2, 5, 6, 8]],
+      [5, [6, 8]],
+      [6, []],
+    ]);
+    expect(computeProfiles(conf1, [])).toEqual(result);
+    expect(computeProfiles(conf2, [])).toEqual(result);
+
+    conf1 = ["F7:G8", "C3:E5"];
+    conf2 = ["C3:E5", "F7:G8"];
+    result = new Map<number, number[]>([
+      [0, []],
+      [2, [2, 5]],
+      [5, [6, 8]],
+      [7, []],
+    ]);
+    expect(computeProfiles(conf1, [])).toEqual(result);
+    expect(computeProfiles(conf2, [])).toEqual(result);
+
+    conf1 = ["G7:H8", "C3:E5"];
+    conf2 = ["C3:E5", "G7:H8"];
+    result = new Map<number, number[]>([
+      [0, []],
+      [2, [2, 5]],
+      [5, []],
+      [6, [6, 8]],
+      [8, []],
+    ]);
+    expect(computeProfiles(conf1, [])).toEqual(result);
+    expect(computeProfiles(conf2, [])).toEqual(result);
+  });
+});
+
+describe("recomputeZones new tests", () => {
```

Comment:
splitting tests in `new` and `old` tests dones't make that much sense ? You can keep the old tests in a commit to make sure your REF didn't break anything, but there should probably be a follow-up commit that merges the new and old tests. 

If we come back here in a year, the distinction of `new/old` tests won't make sense

Reply:
it was an oversight